sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryParsingErrors.scala

```diff
@@ -88,6 +88,11 @@ object QueryParsingErrors {
     new ParseException("DISTRIBUTE BY is not supported", ctx)
   }
 
+  def transformNotSupportQuantifierError(ctx: ParserRuleContext): Throwable = {
+    new ParseException("TRANSFORM not support quantifier DISTINCT/ALL " +
```

Comment:
not support -> does not support

Reply:
`quantifier DISTINCT/ALL` -> `DISTINCT/ALL`

Reply:
Done